### PR TITLE
Cypress Testing v12.8.1 - Notebooks fully working

### PR DESCRIPTION
**Pull request by TackAdam**
### Description
[Describe what this change achieves]
Fixed all of the testing in notebooks.
Greatly improved speed removing unnecessary waits.

### Issues Resolved
[List any issues this PR will resolve]

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by TackAdam** [comment]
Screenshot 2023-05-19 at 9 39 52 AM
All 36 test passing.

File: .cypress/integration/2_notebooks.spec.js [hunk 1]
```diff
@@ -177,55 +146,46 @@ describe('Testing notebooks table', () => {
...
     cy.get('input.euiFieldText').type(TEST_NOTEBOOK);
     cy.get('.euiButton__text')
       .contains(/^Create$/)
       .click();
-    cy.wait(delay * 2);
   });
 });
 
 describe('Test reporting integration if plugin installed', () => {
   beforeEach(() => {
```
Comment (TackAdam) [inline .cypress/integration/2_notebooks.spec.js:157]: Screenshot 2023-05-19 at 9 43 58 AM
Installed reporting integration plugin and running "yarn osd bootsrap" encountered the above error. Adjusting the package.json file allowed it to run successfully.

Comment (TackAdam) [reply .cypress/integration/2_notebooks.spec.js:157]: https://github.com/opensearch-project/dashboards-reporting/blob/main/package.json

**Review by joshuali925 — APPROVED** [review]
thanks

**Review by anirudha — APPROVED** [review]

**Pull request merged**